buildSrc/src/main/java/com/osacky/flank/gradle/FladlePluginDelegate.kt

```diff
@@ -65,6 +66,9 @@ class FladlePluginDelegate {
   private fun TaskContainer.createTasksForConfig(base: FlankGradleExtension, config: FladleConfig, project: Project, name: String) {
     checkIfSanityAndValidateConfigs(config)
     validateOptionsUsed(config = config, flank = base.flankVersion.get())
+    val configName = name.toLowerCase()
+    val useDefaultDir = config.localResultsDir.isPresent.not() && configName.isNotBlank()
+    val setUpWorkingDir: JavaExec.() -> Unit = { if (useDefaultDir) workingDir(project.layout.buildDirectory.dir("fladle/$configName")) }
```

Comment:
woah is this a function declared in line? never seen that before, but since we have control of the `FlankJavaExec` task, can we just define the function in there?

Reply:
Hm...sure I can rewrite it, my intention was, a function that is used to set up working dir is created for each configuration (wraps some params in the scope)

Reply:
> otherwise, i think this makes sense. we'll need some tests for this though since i'm getting quite confused with all the conditional logic :)

That could be hard (at least for me). I don't know how to programmatically verify if working directory is correctly set. Another approach would be to invoke `runFlank[config] -PdumpShards` and check if json is saved into correct dir but that would require CI to be authenticated...